topics/metagenomics/tutorials/plasmid-metagenomics-nanopore/tutorial.md

```diff
@@ -0,0 +1,514 @@
+a	utg000001l	0	channel_364_204a2254-2b6f-4f10-9ec5-6d40f0b870e4_template:101-4457	+	4357
+```
+
+## Remapping using Minimap2
+
+The Assembly graph created can be used for mapping again with minimap2, but first the graph should be transformed to FASTA format. Remapping is done on the miniasm assembly to map the original reads on the assembly to provide the nucleotides per position. This is used by **Racon** {% icon tool %} for consensus construction.
```

Comment:
```suggestion
The Assembly graph created can be used for mapping again with minimap2, but first the graph should be transformed to FASTA format. Remapping is done on the miniasm assembly to map the original reads on the assembly to provide the read depth at each position. This is used by **Racon** {% icon tool %} for consensus construction.
```



not sure about read depth, think that's what's being described here or?

Reply:
It's to improve the consensus base call, I rephrased for clarity, thanks!